models/question.js

```diff
@@ -7,10 +7,10 @@ const questionSchema = new Schema({
   choiceB: { type: String, required: false },
   choiceC: { type: String, required: false },
   choiceD: { type: String, required: false },
-  answer: { type: String, required: true },
-  quizID: { type: Schema.Types.ObjectId, ref: "Quiz" }
+  answer: { type: String, required: false },
+  quiz: { type: Schema.Types.ObjectId, ref: "Quiz" }
```

Comment:
I switch to false for now, but once we have the choices and answer input we can make this a requirement before submitting within mongo